python/ray/train/_internal/results_preprocessors/aggregate/aggregate_fn.py

```diff
@@ -1,129 +1,9 @@
-import abc
-from typing import Dict, List, Union, Optional
-import warnings
-
-import numpy as np
-
-from ray.train._internal.results_preprocessors.aggregate.aggregate_utils import (
-    VALID_AGGREGATE_TYPES,
-    _get_weights_from_results,
-)
-from ray.util.annotations import Deprecated
-
-_deprecation_msg = (
+raise DeprecationWarning(
     "`ray.train.callbacks.results_preprocessors.aggregate` and the `ray.train.Trainer` "
     "API are "
     "deprecated in Ray "
-    "2.0, and is replaced by Ray AI Runtime (Ray AIR). Ray AIR "
+    "2.1, and are replaced by Ray AI Runtime (Ray AIR). Ray AIR "
```

Comment:
Let's keep our deprecation messages as `deprecated in Ray 2.0`. These were all marked as deprecated in 2.0 anyways.